### ISSUE TITLE
Some Important bugs to be fixed and one suggestion

### ISSUE THREAD

**Issue by Ponury2003** [Tsuey/L4D2-Community-Update#455]
### Description

BUGS (Very important yet easily fixable since there are already fixed of these at workshop and should be at vanilla game)

- Francis Rifle animations: this is the most important for me, tls update fixed every single bad animation but this one, it’s weird for me since Francis shares animation with Louis and Louis has it fixed and Francis no. Francis using coach rifle animation is a bug that is leftover from valve lazy porting orginal campaigns to sequel, He always in l4d1 and THE SACRIFICE COMIC held it like Bill and Louis so next to face. Since Francis is my favorite survivor ever and m16 is my favorite weapon it annoys me that in third person he hold it weirdly

- Defibrillator and bile jar at The Sacrifice and No mercy never spawns: It can only spawn in first aid cabinets if survivors have black and white vision and I think it’s big game flaw to have it unavailable at 2 maps for some reason especially since one take place at hospital, bike jars are available from ceda agents infected so it’s less of a problem.

- At blood harvest only 2 l4d1 starting weapons can be, well starting weapons, another small flaw that I believe is easily fixable 

suggestion:

- At L4D1 campaigns and orginal survivors at the passing should use m1911 instead of l4d2 pistols: not bug but if we can have orginal special infected on those maps then pistols should not be a problem as well 

### Additional files

Here are pictures of how Francis use rifle in canon (l4d1, l4d2 fix and comic) so you know how he should do that:
![IMG_0603](https://github.com/Tsuey/L4D2-Community-Update/assets/149505093/7223fe22-0f53-4e70-b6a7-9d3a4bdde55f)
![IMG_0602](https://github.com/Tsuey/L4D2-Community-Update/assets/149505093/c6282133-bb21-448b-910d-eef7a74910a8)
![IMG_0601](https://github.com/Tsuey/L4D2-Community-Update/assets/149505093/85c47699-df3f-4bb3-b48a-940dea302da2)

and here how he do that as unfixed state 

![IMG_0604](https://github.com/Tsuey/L4D2-Community-Update/assets/149505093/9d01d101-1732-43db-8872-6c646d6d5f02)

**Comment by alexiscoutinho** [comment]
I agree that a hospital not having any defibrillators is quite strange.
